ElasticBox/jenkins-elasticbox-setup.md

```diff
@@ -0,0 +1,335 @@
+“date”: “09-01-2016”,
+“author”: “”,
+“attachments”: [],
+“contentIsHTML”: false
+}}}
+
+To continuously integrate and deploy through ElasticBox, you need a Jenkins server set up to work with your source control management system (SCM) and the [ElasticBox plugin](https://wiki.jenkins-ci.org/display/JENKINS/ElasticBox+CI).
+
+If GitHub is your SCM, follow these sections to integrate Jenkins, GitHub plugins, and ElasticBox.
+
+**In this article:**
+
+* Install Jenkins with ElasticBox and Git plugins
+* Connect ElasticBox in Jenkins
+
+### Install Jenkins with ElasticBox and Git Plugins
+
+Follow these steps to build and deploy Jenkins server from a box. The box installs the latest open source Jenkins server with GitHub plugins, dependencies, and the ElasticBox plugin.
+
+**IMPORTANT:** Since the Jenkins server box is Debian based, remember to deploy it on a Debian Ubuntu Linux image.
+
+### Create a Jenkins Server Box
+
+**Steps**
+
+1. On the Boxes page, click **New Box**. Enter these details and save.
+
+	![integrate-jenkins1](../images/ElasticBox/integrate-jenkins1.png)
+
+2. Add the following variables to store Jenkins server deployment values.
+Provide your own Git repository through a text variable called GIT_REPOSITORY_URL.
+
+	![integrate-jenkins2](../images/ElasticBox/integrate-jenkins2.png)
+
+Pass the GitHub access token in a text variable.
+
+![integrate-jenkins3](../images/ElasticBox/integrate-jenkins3.png)
+
+To get it, sign in to your [GitHub](https://github.com/) account. Under settings, go to **Applications** > **Personal access tokens** > **Generate new token**. Describe why you need the token. Under scope, select **repo** and **repo: status**. Then click **Generate Token**. Copy and paste it as shown.
+Provide your GitHub project URL in a text variable called GITHUB_PROJECT_URL.
+
+![integrate-jenkins4](../images/ElasticBox/integrate-jenkins4.png)
+
+Provide your GitHub username as a text variable called GITHUB_USER.
+
+![integrate-jenkins5](../images/ElasticBox/integrate-jenkins5.png)
+
+Open HTTP port 8080 on the Jenkins server through a port variable called HTTP to allow Internet traffic.
+
+![integrate-jenkins6](../images/ElasticBox/integrate-jenkins6.png)
+
+Refer to the ElasticBox, Git, and GitHub plugin dependencies in a text variable called PLUGINS.
+Enter this value:
+
+```
+elasticbox git github
+
+```
+
+![integrate-jenkins7](../images/ElasticBox/integrate-jenkins7.png)
+
+Add a plugin that triggers GitHub to push pull requests.
+
+![integrate-jenkins8](../images/ElasticBox/integrate-jenkins8.png)
+
+Copy, paste this script in a text file, save in XML, and upload to a file variable called GITHUB_PLUGIN_CONFIG.
+
+```
+
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.jenkins.GitHubPushTrigger_-DescriptorImpl plugin="github@1.9.1">
+    <manageHook>true</manageHook>
+    <credentials>
+        <com.cloudbees.jenkins.Credential>
+            <username>{{ GITHUB_USER }}</username>
```

Comment:
All `{{` inside of code blocks need to be escaped by using `\{{`. Fix here and all other places in this file.